src/audio-worklet/migration/spn-recorder/app.js

```diff
@@ -0,0 +1,289 @@
+/**
+ * @license Copyright (c) Google. MIT License.
```

Comment:
We're not using MIT license:
https://github.com/GoogleChromeLabs/web-audio-samples/blob/main/LICENSE

Reply:
Just gonna use the same header as the rest of the files